### PR TITLE
# MVP - Weather Web App

### PR DESCRIPTION
**Pull request by cliftondavies**
### Completed:
- [x] Gitflow
- [x] Github Actions
- [x] ESLint
- [x] Stylelint
- [x] .gitignore
- [x] package.json
- [x] webpack
- [x] ES6
- [x] Error handling for promises or async/await.
- [x] README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^5.9.0",
+    "webpack-cli": "^4.2.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (shloch) [inline package.json:31]: Commands to run the app on top of building the scripts only would be nice to have, something like this :
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

**Review by shloch — APPROVED** [review]
## Hello @cliftondavies 

This is great. It is well built according to specifications.

The code structure is clean, logic and UI are separated and linting is very correct
code

App works smoothly

weather

THUMBS UP

**Comment by shloch** [comment]
You can use my gif image above on your README if you like, it's more descriptive of your work from a first glance

**Comment by cliftondavies** [comment]
## Hi @shloch 👋

### Thanks for the review, and the scripts to add. 👍 🙏 

### The gif looks good also! What tool did you use for it? 😃📝

**Comment by shloch** [comment]
@hello @cliftondavies , i use a MackBook, and the app for the .gif is called `licecap`, there's a windows version I think

**Pull request merged**